tcms/testruns/tests/test_permissions.py

```diff
@@ -0,0 +1,63 @@
+from django.urls import reverse
+from http import HTTPStatus
+
+from tcms import tests
+from tcms.tests import factories
+from tcms.testcases.models import TestCaseStatus
+
+
+class TestCreateTestRun(tests.PermissionsTestCase):
+    permission_label = 'testruns.add_testrun'
+    url = reverse('testruns-new')
+    http_method_names = ['get', 'post']
```

Comment:
The view under test doesn't support GET requests.

Note: I get a 200 in browser instead of 4xx, not sure why ATM.

Reply:
This is still an issue but you failed to see b/c of the missing changes introduced in #1196. Once that is merged rebase and this will start erroring out.